pathz/pathz.proto

```diff
@@ -201,10 +208,23 @@ message UploadRequest {
   uint64 created_on = 3;
 
   // The actual OpenConfig Path-based Authz Policy.
-  // It is provided as a PROTO that is defined by
-  // https://github.com/openconfig/public/release/authorization/authorization.proto.
-  google.protobuf.Any policy = 4;
+  gnsi.authorization.AuthorizationPolicy policy = 4;
 }
 
 message UploadResponse {}
 
+// ProbeRequest contains a single user name and gNMI Path being attempted.
+// Data returned to an RPC Caller should adhere to the policy.
+message ProbeRequest {
+  string user = 1;
+  gnmi.Path path = 2;
+}
```

Comment:
we didn't talk about this directly, but since the policies are versioned, do we want to provide a version here ? (I don't know that much history is expected to be held available on the server side... so maybe version isn't helpful)

Reply:
The `version` field was supposed to be used only to help the administrator to check if the device has the latest version of the policy installed.

I did not plan to keep multiple versions of the same policy beyond the previous one (so it can be restored in the case of trouble with the Rotate() RPC) and new one (the one being uploaded/activated in this Rotate call).
However, the Install() RPC allows for uploading different policies and have them associated with an ID (a string). Perhaps using this ID to specify an inactive policy to evaluate its response is a better approach than a version of 

BTW the activation of a policy is out of scope of this API - the assumption is that it will be done via gNMI.